migrations/20210310224007_remove_dns_confirmation_code/migration.sql

```diff
@@ -7,5 +7,5 @@
 
 */
 ALTER TABLE "sites" DROP COLUMN "dns_tag",
-DROP COLUMN "verified",
-DROP INDEX "dns_tag_unique";
+DROP COLUMN "verified";
+DROP INDEX IF EXISTS "dns_tag_unique";
```

Comment:
Hey, these migrations are autogenerated. Was there an issue when you tried running these?

Reply:
Yes migrations failed, the syntax is wrong dropping an index is a global operation and doesn't fit inside an alter table statement

Reply:
after researching a bit more turns out when u drop a column in postgres, the index is automatically dropped, probably the reason i had to use `if exists` then